corehq/apps/app_manager/static/app_manager/js/case_config_utils.js

```
@@ -1,5 +1,7 @@
 "use strict";
 hqDefine('app_manager/js/case_config_utils', function () {
+    const sum = "10" + 5;
```

ESLint

[error] 'sum' is assigned a value but never used.

(no-unused-vars)